.eslintrc

```diff
@@ -0,0 +1,11 @@
+{
+  "parser": "babel-eslint",
+  "rules": {
+    "strict": 0,
```

Comment:
[the docs](http://eslint.org/docs/rules/strict) say that this property only accepts string values